CHANGELOG_PENDING.md

```diff
@@ -5,3 +5,6 @@
 ### SDK Enhancements
 
 ### SDK Bugs
+* `aws/session`: Fix SDK AWS_PROFILE and static environment credential behavior ()
+  * Fixes the SDK's behavior when determining the source of credentials to load. Previously the SDK would ignore the AWS_PROFILE environment, if static environment credentials were also specified.
```

Comment:
Should this be a SDK bug or an Enhancement, as it would be a minor release?

Reply:
Bug in this case, because it was incorrect, unexpected, behavior in the SDK.